pandas/core/tools/timedeltas.py

```diff
@@ -64,6 +69,23 @@ def to_timedelta(arg, unit='ns', box=True, errors='raise'):
     TimedeltaIndex(['0 days', '1 days', '2 days', '3 days', '4 days'],
                    dtype='timedelta64[ns]', freq=None)
 
+    Returning an ndarray by using the 'box' keyword argument:
+
+    >>> pd.to_timedelta(np.arange(5), box=False)
+    array([0, 1, 2, 3, 4], dtype='timedelta64[ns]')
+
+    Add new column of dates from existing dates in a `DataFrame`
```

Comment:
I'd remove this example as well; it demonstrates assignment more than `to_timedelta`

Reply:
The one below it demonstrating DataFrame column assignment